### DISCUSSION TITLE
Detailed changelog

### DISCUSSION THREAD

**Discussion by diegolmello** [react-navigation/react-navigation#9162] [Ideas]
Hey, everyone.
I don't have a solution here, but it'd be awesome to have a changelog separated by package.
Currently, when a version is released for one package, all of them are released with a version bump.
So if we want to know what changed for a specific package after a couple of versions, we have to open 4 or 5 release pages.
What do you think?
Thank you.

**Comment by satya164** [comment]
That's a limitation due to way Lerna handles publishing. I'm working on an alternate solution which will avoid that, but it's gonna take a while.

**Reply by diegolmello** [reply]
Awesome. Thank you!
